dockerfiles/container-registry-agent/Dockerfile.ubi8

```diff
@@ -0,0 +1,63 @@
+FROM snyk/ubuntu as base
+
+MAINTAINER Snyk Ltd
+
+USER root
+
+RUN apt update && apt install -y make python gcc
+
+RUN apt-get update && apt-get install -y ca-certificates
+
+ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
+
+ENV PATH=$PATH:/home/node/.npm-global/bin
+
+RUN npm install --global snyk-broker
+
+
+
+FROM registry.access.redhat.com/ubi8/nodejs-16-minimal
+USER root
+
+ENV PATH=$PATH:/home/node/.npm-global/bin
+
+COPY --from=base /home/node/.npm-global /home/node/.npm-global
+
+COPY --from=base /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+
+# Don't run as root
+RUN mkdir -p /home/nobody && chown nobody:nobody /home/nobody
+WORKDIR /home/nobody
+USER nobody
+
+# Prepare image entrypoint
+COPY --chown=nobody:nobody ./bin/container-registry-agent/docker-entrypoint.sh ./docker-entrypoint.sh
```

Comment:
`docker-entrypoint.sh` uses `/bin/ash` for alpine which shouldn't be available in ubi8. May need a version that uses `/bin/bash`.

Reply:
Indeed, there's none. Fixed it by creating a symlink to /bin/bash specifically for the RedHat based image, so that we do not have to modify the docker-entrypoint.sh script. Thanks for noticing it.